certbot-apache/certbot_apache/configurator.py

```diff
@@ -868,7 +874,25 @@ def _sift_rewrite_rule(self, line):
         # Sift line if it redirects the request to a HTTPS site
         return target.startswith("https://")
 
-    def _copy_create_ssl_vhost_skeleton(self, avail_fp, ssl_fp):
+    def _section_blocks(self, blocks):
+        out = []
+        while len(blocks) > 1:
+            start = blocks[0]
+            end = blocks[1] + 1
+            out += range(start, end)
+            blocks = blocks[2:]
+        return out
+
+    def _create_block_segments(self, orig_file_list, vhost_num):
+        blocks = [idx for idx, line in enumerate(orig_file_list)
+                     if line.lstrip().startswith("<VirtualHost")
+                     or line.lstrip().startswith("</VirtualHost")]
```

Comment:
To catch lowercase instances `line.lower().lstrip().startswith("</virtualhost")`